annotation-file-utilities/src/annotator/find/IsSigMethodCriterion.java

```diff
@@ -403,6 +382,16 @@ public boolean isSatisfiedBy(TreePath path) {
     return true;
   }
 
+  /** Find a type parameter's bound in the current context. */
+  private String erasure(TypeParameterTree param) {
```

Comment:
"Erasure" also means dropping type parameters as in Map<String, Integer> gets erased to Map.  Would a name that relates to "bound" be better than "erasure"?